File: PHP/Account.php
<?php

class Account {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name, $document){
        $this->name = $name;
        $this->document = $document;
    }

    public function printData(){
        var_dump("Name: ".$this->name.", Document: ".$this->document);
    }
}

File: Python/account.py
# creación de clase para la notación es lowercamelcase y para las clases upper
class Account:
    id          = int
    name        = str
    document    = str
    email       = str
    password    = str
    
    # DEFINIMOS EL CONSTRUCTOR
    def __init__(self, name, document):
        self.name       = name
        self.document   = document